tests/transactional/check_phub.pm

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test packagehub repo on sle micro
+#
+# 1. Check if PackageHub extention is availble on sle micro
+# 2. Activate PackageHub extention
+# 3. Install a package from PackageHub repo
+#
+# Maintainer: qac <qa-c@suse.de>
+
+package check_phub;
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use version_utils 'is_sle_micro';
+use transactional;
+
+sub run {
+    select_console 'root-console';
+    my $out = script_output("transactional-update --quiet register --list-extensions");
+    if ($out =~ m#PackageHub/(.*)/#) {
+        trup_call("register -p PackageHub/$1/" . get_var('ARCH'));
+        process_reboot(trigger => 1);
+        trup_call("pkg install sshpass");
+        process_reboot(trigger => 1);
+        validate_script_output("rpm  -qa|grep sshpass", sub { m/sshpass/ });
+    }
+    else {
+        record_info('AVC', "PackageHub Unavaible", result => 'softfail');
```

Comment:
`s/AVC/info/`, so we dont confuse this message with SELinux messages. :)

Reply:
shouldn't we actually `die` instead of softfail? The purpose of the test is to see if we can install a package from phub

Reply:
I was thinking of this as well. I guess if the test is not suitable for some version, let's just don't schedule it

Reply:
This test is to check if phub extension is avaiable in repository server (proxy.scc.suse.de) provided by openQA. It's currently all-21.2.proxy.scc.suse.de. The server name might changes but we can take them as one server because of the the same scc code. 
The phub is not enabled in early developemt stage until (maybe) after Beta as @waynechen55  mentioned above. If we use "die" instead of "softfail", the test including module will always fail until phub is available.

Reply:
sle micro is generally behind of sles schedule, so the alpha phase of phub should not be an issue over here.

Reply:
Hello guys, 
Which option would you like:
1.  die,  the test will fail until phub is available in proxy.
2. record_info,  record_info('AVC') is more noticeable/highlighted than record_info('info') .  It would be easier for test reviewer to review.  

I am fine with either of option.  You are the "Latest test" reviewers.   It's your call :)

Reply:
Again, Phub won't be available using SCC proxy, because it's not a module which is delivered together with SLE Micro, so it is not synchronized with SLE Micro images. So, we can only test it using real SCC registration. 

I prefer record_info, but as Martin said, `AVC` is for SELinux related messages, doesn't make sense to mix SELinux with Phub.